src/smp.c

```diff
@@ -18,6 +18,7 @@
 #define CPU_START_OFF_T8103    0x54000
 #define CPU_START_OFF_T8112    0x34000
 #define CPU_START_OFF_T6020    0x28000
+#define CPU_START_OFF_T6030    0x34000
```

Comment:
We didn't redefine offsets for T600x, so using `CPU_START_OFF_T8112` for T6030 would be more consistent